src/context.js

```diff
@@ -6,6 +6,13 @@ var TEXT = require('./text');
 
 var Point = require('./Point');
 
+const PATH_COMMAND = {
+    MOVE: 'm',
+    LINE: 'l',
+    QUADRATIC_CURVE: 'q',
+    BEZIER_CURVE: 'b'
+};
+
```

Comment:
Just a note to say that this is likely to conflict with 3d87dcbbd0c707b26c2dfa02a0278a647c0ab1f8 as it adds code immediately above `class Context`.